pkg/kubectl/cmd/create_deployment.go

```diff
@@ -153,7 +165,18 @@ func createDeployment(f cmdutil.Factory, cmdOut, cmdErr io.Writer,
 	generatorName = fallbackGeneratorNameIfNecessary(generatorName, resourcesList, cmdErr)
 
 	imageNames := cmdutil.GetFlagStringSlice(cmd, "image")
-	generator, ok := generatorFromName(generatorName, imageNames, deploymentName)
+	replicas := cmdutil.GetFlagInt(cmd, "replicas")
+
+	command := []string{}
+	argsArray := args[1:]
```

Comment:
I'm not sure if this is the final pattern anyway since I'm pending a response to my previous comment.

> @deads2k with regard to --command and the -- separator: do you think it is worth introducing the -- separator syntax when it would make kubectl create inconsistent with kubectl run? I personally like the -- syntax better but I tend to value consistency above all else.